src/outagesim/backup_reliability.jl

```diff
@@ -434,20 +697,21 @@ Return a dictionary of inputs required for backup reliability calculations.
     -gen_operational_availability::Real = 0.9998        Fraction of year generators not down for maintenance
     -gen_failure_to_start::Real = 0.0066                Chance of generator starting given outage
     -gen_failure_to_run::Real = 0.00157                 Chance of generator failing in each hour of outage
-    -num_gens::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
-    -gen_capacity::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
+    -num_gen::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
+    -gen_capacity_kw::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
```

Comment:
Why is there this 0.1 cut off? Why not make the default nothing, then use gen_capacity_kw if not nothing and the REopt results value otherwise?